backend/substrapp/compute_tasks/image_builder.py

```diff
@@ -114,6 +115,16 @@ def _get_entrypoint_from_dockerfile(dockerfile_dir: str) -> list[str]:
     raise compute_task_errors.BuildError("Invalid Dockerfile: Cannot find ENTRYPOINT")
 
 
+def _delete_kaniko_pod(create_pod: bool, k8s_client: kubernetes.client.CoreV1Api, pod_name: str) -> str:
+    logs = ""
+    if create_pod:
+        logs = get_pod_logs(k8s_client, pod_name, KANIKO_CONTAINER_NAME, ignore_pod_not_found=True)
+        delete_pod(k8s_client, pod_name)
+        for line in (logs or "").split("\n"):
+            logger.info(line, pod_name=pod_name)
```

Comment:
Do we want to log this line by line, rather than as a single JSON line?

Reply:
Well spotted, that might be the very reason why we have _so many_ lines of log coming from Kaniko.
I'd be in favour to try adding all in the same log, what do you think @guilhem-barthes ?